client/src/components/imagine25/pages/GalagaInstruction.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Frame } from "../components/Frame";
+import { navigate } from "@reach/router";
+
+const GalagaInstructions = () => {
+  const fontSize = "xs:tw-text-md xl:tw-text-xl";
+  return (
+    <>
+      <h3>Instructions</h3>
+      {Frame(
+        <div className="xs:tw-pt-5 md:tw-pt-8 lg:tw-pt-10">
+          <p className={fontSize}>
+            You will be playing against your selected opponent to see who can
+            score the most points in 1 minute without dying. Your teammate will
```

Comment:
nit: it looks like you can continue playing the game even if you die, so i would remove the part of the sentence that says "without dying" -> "You will be playing against your selected opponent to see who can score the most points in 1 minute"